lib/widgets/selection/unified_toggle.dart

```diff
@@ -0,0 +1,501 @@
+                    ],
+                  ],
+                ),
+              ),
+              const SizedBox(width: 16),
+              toggle,
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+/// Helper widget for form contexts that need FormBuilder integration
+/// Preserves exact FormSwitch behavior and styling
+class UnifiedFormBuilderToggle extends StatelessWidget {
+  const UnifiedFormBuilderToggle({
+    required this.name,
+    required this.title,
+    this.initialValue,
+    this.subtitle,
+    this.variant = UnifiedToggleVariant.normal,
+    this.activeColor,
+    this.enabled = true,
+    this.onChanged,
+    this.validator,
+    this.semanticLabel,
+    this.semanticsLabel,
+    super.key,
+  });
+
+  /// Form field name
+  final String name;
+
+  /// Initial value
+  final bool? initialValue;
+
+  /// Main label text
+  final String title;
+
+  /// Optional subtitle/description text
+  final String? subtitle;
+
+  /// Visual variant of the toggle
+  final UnifiedToggleVariant variant;
+
+  /// Custom active color (overrides variant color)
+  final Color? activeColor;
+
+  /// Whether the toggle is enabled
+  final bool enabled;
+
+  /// Optional change callback
+  final ValueChanged<bool?>? onChanged;
+
+  /// Optional validator
+  // ignore: avoid_positional_boolean_parameters
+  final String? Function(bool?)? validator;
+
+  /// Semantic label for accessibility (old naming)
+  final String? semanticLabel;
+
+  /// Semantic label for accessibility (FormSwitch naming)
+  final String? semanticsLabel;
+
+  @override
+  Widget build(BuildContext context) {
+    return FormField<bool>(
+      initialValue: initialValue ?? false,
+      validator: validator,
+      builder: (FormFieldState<bool> field) {
+        return Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            UnifiedToggleField(
+              value: field.value ?? false,
+              onChanged: enabled
+                  ? (bool value) {
+                      field.didChange(value);
+                      onChanged?.call(value);
+                    }
+                  : null,
+              title: title,
+              subtitle: subtitle,
+              variant: variant,
+              activeColor: activeColor,
+              enabled: enabled,
+              semanticsLabel: semanticsLabel ?? semanticLabel,
+            ),
+            if (field.hasError)
+              Padding(
+                padding: const EdgeInsets.only(left: 16, top: 4),
+                child: Text(
+                  field.errorText!,
+                  style: TextStyle(
+                    color: context.colorScheme.error,
+                    fontSize: 12,
+                  ),
+                ),
+              ),
+          ],
+        );
+      },
+    );
+  }
+}
+
+/// AI-specific toggle field that matches the exact AiSwitchField styling
+class UnifiedAiToggleField extends StatelessWidget {
```

Comment:
let's have the same toggle everywhere, no need for another variant for the AI pages